include/walker-dropdown.php

```diff
@@ -22,9 +22,10 @@ class PLL_Walker_Dropdown extends Walker {
 	function start_el( &$output, $element, $depth = 0, $args = array(), $current_object_id = 0 ) {
 		$value = $args['value'];
 		$output .= sprintf(
-			"\t" . '<option value="%1$s"%2$s%3$s>%4$s</option>' . "\n",
+			"\t" . '<option value="%1$s"%2$s%3$s%4$s>%5$s</option>' . "\n",
 			esc_attr( $element->$value ),
 			method_exists( $element, 'get_locale' ) ? sprintf( ' lang="%s"', esc_attr( $element->get_locale( 'display' ) ) ) : '',
+			( ! empty( $element->flag_url ) ) ? sprintf( ' flag="%s"', esc_attr( $element->flag_url ) ) : '',
```

Comment:
@Chouby I could add an `is_admin()` check in there so the flag url's only get added in the WP admin. I understand that this walker is used on the frontend as well.

Reply:
@Chouby please let me know if you want the `is_admin()` check in there.

Reply:
See my global comment https://github.com/polylang/polylang/pull/257#issuecomment-420930371